libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_common.c

```diff
@@ -584,6 +734,9 @@ void IotTestBleHal_CreateServiceA()
 void IotTestBleHal_CreateServiceB()
 {
     prvCreateService( &_xSrvcB );
+#if ENABLE_TC_ADD_INCLUDED_SERVICE
+    prvCreateIncludedService( &_xSrvcB, bletestATTR_INCLUDED_SERVICE );
```

Comment:
Should included service be secondary service ( service C ) ?

Reply:
An included service can also be a primary service.